docs/en/02_Developer_Guides/14_Files/01_Image.md

```diff
@@ -18,60 +18,81 @@ Allows upload of images through limiting file extensions with `setAllowedExtensi
 
 See [Topics: Rich Text Editing](/topics/rich-text-editing).
 
-### Resizing Images in PHP
+### Manipulating images in Templates
 
-The following are methods defined on the GD class which you can call on Image Objects. Note to get the following to work
-you need to have GD2 support in your PHP installation and because these generate files you must have write access to
-your tmp folder. 
+You can manipulate images directly from templates to create images that are
+resized and cropped to suit your needs.  This doesn't affect the original
+image or clutter the CMS with any additional files, and any images you create
+in this way are cached for later use. In most cases the pixel aspect ratios of
+images are preserved (meaning images are not stretched).
+
+Here are some examples, assuming the `$Image` object has dimensions of 200x100px:
 
 	:::php
-	// manipulation functions
-	$image->resize(width,height); // Basic resize, just skews the image
-	$image->resizeRatio(width,height) // Resizes an image with max width and height
-	$image->paddedResize(width,height) // Adds padding after resizing to width or height.
-	$image->croppedImage(width,height) // Crops the image from the centre, to given values.
-	$image->resizeByHeight(height) // Maximum height the image resizes to, keeps proportion
-	$image->resizeByWidth(width) // Maximum width the image resizes to, keeps proportion 
-	$image->greyscale(r,g,b) // alters image channels ===
+	// Scaling functions
```

Comment:
This isn't actually `:::php` format code, so do you want to change the doccode to `:::ss`?